package.json

```diff
@@ -27,11 +27,13 @@
     "eslint": "8.47.0",
     "eslint-config-next": "13.4.13",
     "next": "13.4.13",
+    "next-themes": "^0.2.1",
     "nookies": "^2.5.2",
     "postcss": "8.4.27",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-feather": "^2.0.10",
+    "react-icons": "^4.11.0",
```

Comment:
react icons is bit of a heavy library, can you use the existing one.

Reply:
done